src/tests/test_bootstrap_in_place.py

```diff
@@ -279,6 +279,19 @@ def waiting_for_installation_completion(
             if not skip_logs:
                 self.log_collection(master_ip)
 
+    def inject_bootstrap(self, ignition_filename: string, butane_config: string):
+        expected_ignition_name = "bootstrap_initial.ign"
+        shutil.move(ignition_filename, expected_ignition_name)
+        flags = shlex.split(f"--rm -v {os.getcwd()}:/data -w /data")
+        # retry to avoid occassional quay hiccups
+        self.retrying_run_container(
```

Comment:
This will merge the butane_config into the existing content of ignition_filename?
Seems to me that it will override it no?